README.md

```diff
@@ -94,16 +104,18 @@ label-studio
 conda create --name label-studio
 conda activate label-studio
 conda install psycopg2
-pip install label-studio
+pip install poetry
+poetry install label-studio
```

Comment:
These instructions don't need to change. The package is still published on pypi and can be installed with pip